stopwatch/src/app/components/stopwatch/stopwatch.component.ts

```diff
@@ -0,0 +1,78 @@
+import { Component, ElementRef, OnDestroy, ViewChild } from '@angular/core';
+import { buffer, debounceTime, filter, fromEvent, interval, Subject, takeUntil, tap } from 'rxjs';
+
+/**
+ * Responsible for drawing timer component
+ */
+@Component({
+  selector: 'app-stopwatch',
+  templateUrl: './stopwatch.component.html',
+  styleUrls: ['./stopwatch.component.scss'],
+})
+export class StopwatchComponent implements OnDestroy {
+  /**
+   * Reference for the wait button in the markup
+   */
+  @ViewChild('pauseButton', { static: false, read: ElementRef })
+  waitButton!: ElementRef;
+
+  /** Timer initial values */
+  timerSeconds = 0;
```

Comment:
Maybe we can merge timerSeconds and  private timerSource$ = interval(1000); into one stream